src/shell/CMakeLists.txt

```diff
@@ -178,9 +178,11 @@ FOREACH(T ${LEANTESTS})
 ENDFOREACH(T)
 
 # Create a lake test for each subdirectory of `lake` which contains a `test.sh` file.
+# Disable tests that are prone to flake by construction
+# (e.g. laketest_116 has a 'sleep()' for a fixed duration).
```

Comment:
```suggestion
# bootstrap: too slow
# 104, git: not adapted to living in the lean4 repo yet
# 116: non-deterministic (`sleep`)
```

Reply:
@Kha Oh, one thing I had not noticed before  - why is bootstrap considered "too slow"?

Reply:
I don't know where the cut-off for "too slow" should be, but it is certainly much slower than all other tests
```
1692/1706 Test #1686: leaninteractivetest_stdOutput.lean ........................   Passed    5.02 sec
1695/1706 Test #1690: leanlaketest_deps .........................................   Passed    5.49 sec
1700/1706 Test #1693: leanlaketest_init .........................................   Passed    5.52 sec
 737/1706 Test  #417: leantest_notationDelab.lean ...............................   Passed    6.42 sec
1636/1706 Test #1607: leanbenchtest_liasolver.lean ..............................   Passed    6.61 sec
1326/1706 Test #1158: leanruntest_lcnf4.lean ....................................   Passed    6.78 sec
1156/1706 Test  #907: leanruntest_constProp.lean ................................   Passed    7.56 sec
1074/1706 Test  #823: leanruntest_aStructPerfIssue.lean .........................   Passed    8.52 sec
1530/1706 Test #1293: leanruntest_printDecls.lean ...............................   Passed    9.03 sec
1703/1706 Test #1691: leanlaketest_ffi ..........................................   Passed    9.17 sec
 552/1706 Test  #164: leantest_PPRoundtrip.lean .................................   Passed    9.81 sec
1490/1706 Test #1216: leanruntest_meta2.lean ....................................   Passed   10.38 sec
1704/1706 Test #1696: leanlaketest_targets ......................................   Passed   12.47 sec
1705/1706 Test #1689: leanlaketest_bootstrap ....................................   Passed   22.74 sec
```

Reply:
@Kha Ah, yeah. Admittedly, that is expected as it is rebuilding all of Lake. I think of it kind of like a Lean stage2 build but for Lake.